src/CabanaPD_ForceModels.hpp

```diff
@@ -173,7 +175,18 @@ struct BaseTemperatureModel<TemperatureDependent, TemperatureType>
                           const double _temp0 )
         : alpha( _alpha )
         , temp0( _temp0 )
-        , temperature( _temp ){};
+        , temperature( _temp )
+    {
+    }
+
+    // FIXME: use the first model temperature for now.
```

Comment:
This field has to be identical across models since it's the particle field. Of course that's not great since it's repeated across all models